modules/contact/src/userobjects/PenaltyWeightedGapUserObject.C

```diff
@@ -64,17 +64,10 @@ PenaltyWeightedGapUserObject::PenaltyWeightedGapUserObject(const InputParameters
     _use_physical_gap(getParam<bool>("use_physical_gap")),
     _aux_lm_var(isCoupled("aux_lm") ? getVar("aux_lm", 0) : nullptr)
 {
-  auto check_type = [this](const auto & var, const auto & var_name)
-  {
-    if (!var.isNodal())
-      paramError(var_name,
-                 "The displacement variables must have degrees of freedom exclusively on "
```

Comment:
I missed why this piece of code is removed.

Reply:
Because the same check is already present in the parent class.

Reply:
Oh wait.... Did I remove it in the wrong class.. I might have...

Reply:
Fixed. By teh way we've been working hard to move away from `disp_[xyz]` parameters in favor of the `displacements` parameter. Any reason that pattern wasn't used here?

Reply:
The only reason is that we repeated old patterns in these objects. So it can be migrated to newer displacements parameter. In fact, that's a task of a contact maintenance issue for the entire module.